all data/src/battlepass_runner.py

```
@@ -0,0 +1,131 @@
+
+def click_completed_missions():
+    """
+    Recursively clicks all completed missions
+    
+    Note, does not work for seasonal ones. I am too lasy to implement the scrolling for something that is done once a month
+    """
+    
+    logger.info(f"Clicking daily missions")
+    
+    while common.element_exist("pictures/CustomAdded1080p/battlepass/completed_mission.png", threshold=0.9):
+        logger.info(f"Completed mission found")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)